src/server/game/Entities/Item/ItemTemplate.h

```diff
@@ -21,10 +21,16 @@
 
 #include "Common.h"
 #include "SharedDefines.h"
-#include "WorldPacket.h"
 #include <vector>
 
 class ObjectMgr;
+class WorldPacket;
+
+namespace WorldPackets {
+    namespace Query {
+        class QueryItemSingleResponse;
```

Comment:
codestyle

Reply:
I'm not worried about semantics atm, the core structure of the code is the main concern but thanks for the heads up. ^_^